AddQtIosApp.cmake

```diff
@@ -155,7 +158,7 @@ function(add_qt_ios_app TARGET)
     endif()
 
     # Allow user to override QT_IOS_ITS_ENCRYPTION_EXPORT_COMPLIANCE_CODE from cache/command line
-    if(NOT QT_IOS_ITS_ENCRYPTION_EXPORT_COMPLIANCE_CODE)
+    if(QT_IOS_ITS_ENCRYPTION_EXPORT_COMPLIANCE_CODE)
```

Comment:
I believe `if(NOT QT_IOS_ITS_ENCRYPTION_EXPORT_COMPLIANCE_CODE)` is correct.
The goal is too be able to develop without it, don't hardcode it in CMakeLists.
Then in Ci for example you can add `cmake -DQT_IOS_ITS_ENCRYPTION_EXPORT_COMPLIANCE_CODE=123345678765434`
I don't have such code, so i don't know how it is expected to look like

Reply:
My mistake, you're right! Must have had QT_IOS_ITS_ENCRYPTION_EXPORT_COMPLIANCE_CODE cached which made me confused.

Reply:
Yes, as you see quite lots of variable are intended to be used that way.
Apple make it quite hard for dev ahah
![image](https://user-images.githubusercontent.com/17255804/101057779-b5663500-358c-11eb-8840-8f92d00feadf.png)